tests/__init__.py

```diff
@@ -37,8 +37,10 @@
 PREFIX = environ.get('DJ_TEST_DB_PREFIX', 'djtest')
 
 conn_root = dj.conn(**CONN_INFO_ROOT)
+conn_root.query("DROP USER IF EXISTS 'datajoint'@'%%';") 
 conn_root.query("CREATE USER 'datajoint'@'%%' IDENTIFIED BY 'datajoint';")
 conn_root.query("GRANT ALL PRIVILEGES ON `djtest%%`.* TO 'datajoint'@'%%';")
+conn_root.query("DROP USER IF EXISTS 'djview'@'%%';") 
 conn_root.query("CREATE USER 'djview'@'%%' IDENTIFIED BY 'djview';")
 conn_root.query("grant select on `djtest%%`.* to 'djview'@'%%';")
```

Comment:
To update for compatibility between MySQL 5.6/5.7/8, can replace Lines 40-45 with: 

```
# create user if necessary on mysql8
if StrictVersion(conn_root.query(
        "select @@version;").fetchall()[0][0]) >= StrictVersion('8.0.0'):
    conn_root.query("CREATE USER IF NOT EXISTS 'datajoint'@'%%';")
    conn_root.query("CREATE USER IF NOT EXISTS 'djview'@'%%';")

# grant permissions and create users for mysql5.6/5.7
conn_root.query("GRANT ALL PRIVILEGES ON `djtest%%`.* \
        TO 'datajoint'@'%%' IDENTIFIED BY 'datajoint';")
conn_root.query("GRANT SELECT ON `djtest%%`.* \
        TO 'djview'@'%%' IDENTIFIED BY 'djview';")
```

Reply:
ok, included. thanks.